pmd-rules.xml

```diff
@@ -30,6 +30,10 @@
         <exclude name="DataClass" />
         <exclude name="ExcessiveImports" />
         <exclude name="AvoidUncheckedExceptionsInSignatures" />
+        <exclude name="GodClass" />
```

Comment:
What, did you agree to modify these or is it just for testing?

Reply:
As we discussed, for now we will use SuppressWarning annotation to avoid this.